composer.json

```
@@ -14,7 +14,7 @@
   ],
   "autoload": {
     "psr-4": {
-      "Cue\\twig_extensions\\": "src/"
+      "Netzstrategen\\TwigAsset\\": "php/src/"
```

Just /php is perfectly fine and makes it consistent with the /js path.
```suggestion
      "Netzstrategen\\TwigAsset\\": "php/"
```